### DISCUSSION TITLE
Can you please add an "enter" button that is clickable via a trackball.

### DISCUSSION THREAD

**Discussion by pawndini** [Chatterino/chatterino2#4541] [Ideas]
Can you please add an "enter" button that is clickable via a trackball.

Reason: I am a C5 quadriplegic so my fingers don't work and I have to use voice software to type. I can click my trackball with the back of my hand, but I have to bring my on-screen keyboard up, then click on the text field that I have just entered with my voice software, Then click the enter button on my keyboard, Then minimize my on-screen keyboard again.  I'm slow at using my mouse to begin with, but with all these other steps I'm usually unable to post my message in a timely fashion. I also used to have this problem with the browser version of discord, but the PC version has solved that.  Thanks for taking the time to read this :-)

**Comment by Felanbird** [comment]
.... Duplicate of https://github.com/Chatterino/chatterino2/issues/2279

**Reply by pawndini** [reply]
Sorry about the duplicate posts.  I assumed since the first one that I commented on was was dead for over two years It would never be seen again.  Also thanks to Mm2PL for adding the accessibility tag 👍

**Comment by Felanbird** [comment]
Actually, it could be approached from a different way allowing us to set mouse keybinds to the existing hotkeys, as you can already replace the sendMessage hotkey.

**Reply by pawndini** [reply]
Yes, I thought of that.  A friend of mine created a script that used one of my mouse buttons to activate a keyboard key, but it was causing some sort of problem for me (I can't remember what it was now), so I uninstalled it.  But that definitely is a viable workaround.

**Comment by iProdigy** [comment]
Could you please share what voice software you're using?

**Reply by pawndini** [reply]
I am using Nuance Dragon naturally speaking version 13 (I think – it is their last version they made).

**Reply by iProdigy** [reply]
Do voice commands like `Press Enter Key` or `New Line` or `New Paragraph` work?

**Reply by pawndini** [reply]
They don't work for all applications or websites.  This software is a little bit dated if I remember correctly, so I'm not sure how many applications work with it.  Also, I have to switch back and forth between speech mode and command mode… It kind of sucks but it's the best solution I've found so far.

**Reply by pawndini** [reply]
Correction: it is version 15

**Reply by pawndini** [reply]
Unbeknownst to me, the verbal command "press enter" works at twitch TV, discord, and Chatterino!  I think it would still be a good idea to implement an enter button, but for now I have solved my problem and I just want to say thanks to everybody who offered help :-)

**Comment by pajlada** [comment]
In v2.4.4, you will be able to enable a Send button. It will also be available in the Nightly build in about an hour after this message.

It looks like this:
![image](https://user-images.githubusercontent.com/962989/236676076-28b11bf4-8fc2-4c25-9c04-2846ec84251b.png)

It's disabled by default, but can be enabled with the "Show send message button" setting
![image](https://user-images.githubusercontent.com/962989/236676101-14e72d29-826c-453e-a65a-5fcd2837a8aa.png)
